litmus-portal/frontend/src/views/Home/AgentConfiguredHome/AgentInfoContainer/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Paper, Typography } from '@material-ui/core';
+import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward';
+import { ButtonOutlined } from 'litmus-ui';
+import React from 'react';
+import { history } from '../../../../redux/configureStore';
+import {
+  getProjectID,
+  getProjectRole,
+} from '../../../../utils/getSearchParams';
+import useStyles from './styles';
+
+interface AgentInfoContainerProps {
+  agentCount: number;
+}
+
+const AgentInfoContainer: React.FC<AgentInfoContainerProps> = ({
+  agentCount,
+}) => {
+  const classes = useStyles();
+  // const { t } = useTranslation();
+
+  const projectID = getProjectID();
+  const projectRole = getProjectRole();
+
+  return (
+    <div>
+      {/* Agent info container */}
+      <Paper className={classes.agentInfoContainer}>
+        <div className={classes.agentInfoBlock}>
+          <div className={classes.agentInfoData}>
+            <div>
+              <Typography id="agentCount">{agentCount}</Typography>
+              {agentCount === 1 ? (
+                <Typography>Agent</Typography>
+              ) : (
+                <Typography>Agents</Typography>
+              )}
+            </div>
+            <div>
+              <Typography id="agentDesc">
+                Cloud native chaos engineering can be setup on this cluster
+                using Litmus Chaos
+              </Typography>
+            </div>
+          </div>
+        </div>
+        {projectRole === 'Owner' && (
```

Comment:
For any such use (`projectRole==='Owner'`) try to have either have a separate file or declare the constants in the starting so that its easier to make changes later on in one place only.